From 0b4f9a48b46beb08480395fe5e674c9cf79619e6 Mon Sep 17 00:00:00 2001
From: Sandeep Singh <sandeep@freescale.com>
Date: Fri, 30 May 2014 10:44:13 -0400
Subject: [PATCH 380/466] t2080/usdpaa/dts: Change num Rx FQs to 3 for shared
 ethernet

Change num Rx FQs to 3 for shared ethernet.

Signed-off-by: Sandeep Singh <sandeep@freescale.com>
Change-Id: I143c1f42c3ed6b69dafce3f22dbc9640a48f667b
Reviewed-on: http://git.am.freescale.net:8181/13083
Tested-by: Review Code-CDREVIEW <CDREVIEW@freescale.com>
Reviewed-by: Vakul Garg <vakul@freescale.com>
Reviewed-by: Thomas Trefny <Tom.Trefny@freescale.com>
[Original patch taken from QorIQ-SDK-V1.6-SOURCE-20140619-yocto.iso]
Signed-off-by: Bin Jiang <bin.jiang@windriver.com>
---
 arch/powerpc/boot/dts/t2080qds-usdpaa-shared-interfaces.dts |    2 +-
 arch/powerpc/boot/dts/t2080rdb-usdpaa-shared-interfaces.dts |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/powerpc/boot/dts/t2080qds-usdpaa-shared-interfaces.dts b/arch/powerpc/boot/dts/t2080qds-usdpaa-shared-interfaces.dts
index 0709bbd..e016c31 100644
--- a/arch/powerpc/boot/dts/t2080qds-usdpaa-shared-interfaces.dts
+++ b/arch/powerpc/boot/dts/t2080qds-usdpaa-shared-interfaces.dts
@@ -111,7 +111,7 @@
 		ethernet@8 {/* 10G */
 			compatible = "fsl,t2080-dpa-ethernet-shared", "fsl,dpa-ethernet-shared";
 			fsl,bman-buffer-pools = <&bp17>;
-			fsl,qman-frame-queues-rx = <0x5c 1 0x5d 1>;
+			fsl,qman-frame-queues-rx = <0x5c 1 0x5d 1 0x2000 3>;
                         fsl,qman-frame-queues-tx = <0 1 0 1 0x3000 8>;
 		};
 		ethernet@9 {/* 10G */
diff --git a/arch/powerpc/boot/dts/t2080rdb-usdpaa-shared-interfaces.dts b/arch/powerpc/boot/dts/t2080rdb-usdpaa-shared-interfaces.dts
index 21f402f..735dea5 100644
--- a/arch/powerpc/boot/dts/t2080rdb-usdpaa-shared-interfaces.dts
+++ b/arch/powerpc/boot/dts/t2080rdb-usdpaa-shared-interfaces.dts
@@ -111,7 +111,7 @@
 		ethernet@8 {/* 10G */
 			compatible = "fsl,t2080-dpa-ethernet-shared", "fsl,dpa-ethernet-shared";
 			fsl,bman-buffer-pools = <&bp17>;
-			fsl,qman-frame-queues-rx = <0x5c 1 0x5d 1>;
+			fsl,qman-frame-queues-rx = <0x5c 1 0x5d 1 0x2000 3>;
                         fsl,qman-frame-queues-tx = <0 1 0 1 0x3000 8>;
 		};
 		ethernet@9 {/* 10G */
-- 
1.7.10.4

